addons/website/static/src/js/content/ripple_effect.js

```diff
@@ -10,15 +10,6 @@ publicWidget.registry.RippleEffect = publicWidget.Widget.extend({
     },
     duration: 350,
 
-    /**
-     * @override
-     */
-    start: async function () {
-        this.diameter = Math.max(this.$el.outerWidth(), this.$el.outerHeight());
```

Comment:
Can the diameter really change depending on the scroll? Anyways I don't mind checking it each time but:
1) I would add a comment next its computation explaining why
2) Don't remove `this.diameter` from a stable version, continue to update the variable in `this` in case it is used by custo